src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Entities/Enums/RelationshipType.cs

```diff
@@ -130,7 +130,7 @@ public enum RelationshipType
 
     /// <summary>
     /// The from Element treats each to Element as a data file.
-    /// A data file is an artifact that stores data required or optional for the from Element's functionality.
+    /// A data file is an artifact that stores data required or optional for the from Element'ContextsResult functionality.
```

Comment:
The changes in this file appear to be a mistake (I think?)

Reply:
Yes caught all the other ones, but missed this, will remove